Content.Server/Body/Components/BloodstreamComponent.cs

```diff
@@ -154,5 +154,6 @@ public sealed class BloodstreamComponent : Component
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
         public float DrunkTime;
+        public float StutterTime;
```

Comment:
If this always aligns with drunktime shouldn't it just use that?

Reply:
Probably a good idea! They could both use the same variable renamed to something like `statusTime`?

Reply:
SGTM, I think it might also need to be a datafield but I haven't read all of the code around it.

Reply:
I originally made drunktime a datafield in the component, assuming we just drop stuttertime and rename drunktime in both files, should be good!

Reply:
Sounds good, will do